audit/projects/k8s-releng-prod/services/enabled.txt

```diff
@@ -1,2 +1,9 @@
-NAME                     TITLE
-cloudkms.googleapis.com  Cloud Key Management Service (KMS) API
+NAME                              TITLE
+cloudbuild.googleapis.com         Cloud Build API
+cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
+containerregistry.googleapis.com  Container Registry API
+logging.googleapis.com            Cloud Logging API
+pubsub.googleapis.com             Cloud Pub/Sub API
+secretmanager.googleapis.com      Secret Manager API
+storage-api.googleapis.com        Google Cloud Storage JSON API
+storage-component.googleapis.com  Cloud Storage
```

Comment:
So this is fun. FYI @kubernetes/release-engineering 

When running `./infra/gcp/ensure-release-projects.sh` to deploy https://github.com/kubernetes/k8s.io/pull/2333, a _whole bunch of other stuff_ was created.

The problem is that I put `k8s-releng-prod` under `release` in [infra/gcp/infra.yaml](https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/infra.yaml), thinking that it's effectively the same scope/group of people, and so `ensure-release-projects.sh` provisioned it the same as `k8s-release`

I will manually undo all of this, and can setup a separate `releng` group, but maybe now is a good time to ask if it's still necessary to have `k8s-releng-prod` as its own special-purpose project?

Reply:
Discussion in https://kubernetes.slack.com/archives/CJH2GBF7Y/p1626374201181100 led to: going to keep k8s-releng-prod as its own special purpose project, since there are google.com projects that are tangled up with it that are not nearly as easy to change as stuff here

Reply:
I ran the following to undo the oops:
```bash
#!/usr/bin/env bash
# buckets
gsutil rb gs://k8s-releng-prod
gsutil rb gs://k8s-releng-prod-gcb
gsutil rb gs://artifacts.k8s-releng-prod.appspot.com/

# services
gcloud services disable --force cloudbuild.googleapis.com --project=k8s-releng-prod
gcloud services disable --force containerregistry.googleapis.com --project=k8s-releng-prod
gcloud services disable --force logging.googleapis.com --project=k8s-releng-prod
gcloud services disable --force pubsub.googleapis.com --project=k8s-releng-prod
gcloud services disable --force secretmanager.googleapis.com --project=k8s-releng-prod
gcloud services disable --force storage-api.googleapis.com --project=k8s-releng-prod
gcloud services disable --force storage-component.googleapis.com --project=k8s-releng-prod

# project iam bindings
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=serviceAccount:689098785215@cloudbuild.gserviceaccount.com --role=roles/cloudbuild.builds.builder
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com --role=roles/cloudbuild.builds.builder
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-admins@kubernetes.io --role=roles/cloudbuild.builds.editor
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-editors@kubernetes.io --role=roles/cloudbuild.builds.editor
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=serviceAccount:service-689098785215@gcp-sa-cloudbuild.iam.gserviceaccount.com --role=roles/cloudbuild.serviceAgent
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=serviceAccount:service-689098785215@containerregistry.iam.gserviceaccount.com --role=roles/containerregistry.ServiceAgent
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=serviceAccount:service-689098785215@gcp-sa-pubsub.iam.gserviceaccount.com --role=roles/pubsub.serviceAgent
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-admins@kubernetes.io --role=roles/serviceusage.serviceUsageConsumer
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-editors@kubernetes.io --role=roles/serviceusage.serviceUsageConsumer
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-artifact-admins@kubernetes.io --role=roles/viewer
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-editors@kubernetes.io --role=roles/viewer
gcloud projects remove-iam-policy-binding k8s-releng-prod --member=group:k8s-infra-release-viewers@kubernetes.io --role=roles/viewer
```

I verified locally by running audit for k8s-releng-prod and verifying there were no differences compared to the current files in HEAD
```
$ make -C images/k8s-infra TAG=latest run WHAT="./audit/audit-gcp.sh k8s-releng-prod"
$ git diff

Reply:
https://github.com/kubernetes/k8s.io/pull/2349 will prevent this from happening again